### PR TITLE
Add print performance note

### PR DESCRIPTION
**Pull request by sbrunner**
Get from https://github.com/mapfish/mapfish-print/issues/312

### CODE REVIEW

File: doc/developer/server_side.rst [hunk 1]
```diff
@@ -53,6 +53,17 @@ of an existing c2cgeoportal application. Here's how:
 
     rm .build/requirements.timestamp && make -f <user>.mk build
 
+.. note:: Print performance issue
+
+   When we restart the print frequantly we can have performance issue
+   on randum number generation.
```
Comment (ger-benjamin) [inline doc/developer/server_side.rst]: randum -> random

Comment (sbrunner) [inline doc/developer/server_side.rst]: fixed, thanks :-)

File: doc/developer/server_side.rst [hunk 2]
```diff
@@ -53,6 +53,17 @@ of an existing c2cgeoportal application. Here's how:
 
     rm .build/requirements.timestamp && make -f <user>.mk build
 
+.. note:: Print performance issue
+
+   When we restart the print frequantly we can have performance issue
```
Comment (asaunier) [inline doc/developer/server_side.rst]: When restarting the print server frequently, performance issues may randomly be observed.

File: doc/developer/server_side.rst [hunk 3]
```diff
@@ -53,6 +53,17 @@ of an existing c2cgeoportal application. Here's how:
...
+   on random number generation.
+
+   To have better performance you should add in the ``/srv/tomcat/tomcat1/bin/setenv-local.sh`` file:
```
Comment (asaunier) [inline doc/developer/server_side.rst]: To improve the performances

File: doc/developer/server_side.rst (hunk 2 above)
Comment (sbrunner) [inline doc/developer/server_side.rst]: => `When restarting the print server frequently, performance issues may randomly be observed. This is done in random number generation.`?

File: doc/developer/server_side.rst (hunk 3 above)
Comment (sbrunner) [inline doc/developer/server_side.rst]: done :-)

**Comment by ger-benjamin** [comment]
Seems alright but one question:
If that does ``improve the performances`` why this ``export ADD_JAVA_OPTS`` is not added by default ?

**Comment by sbrunner** [comment]
Because it's less secure, if we use the random for performance. The default random generator wait that the entropy is good to generate a random number.

**Comment by ger-benjamin** [comment]
Ok, thanks. LGTM

**Pull request merged**